src/main/java/io/jmix/uisamples/view/flowui/components/virtuallist/inlineeditor/VirtualListInlineEditor.java

```diff
@@ -0,0 +1,195 @@
+package io.jmix.uisamples.view.flowui.components.virtuallist.inlineeditor;
+
+
+import com.vaadin.flow.component.Html;
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.avatar.Avatar;
+import com.vaadin.flow.component.avatar.AvatarVariant;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.dependency.CssImport;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.data.renderer.Renderer;
+import com.vaadin.flow.server.StreamResource;
+import io.jmix.core.LoadContext;
+import io.jmix.core.Messages;
+import io.jmix.core.Resources;
+import io.jmix.flowui.DialogWindows;
+import io.jmix.flowui.kit.action.ActionPerformedEvent;
+import io.jmix.flowui.model.CollectionContainer;
+import io.jmix.flowui.model.CollectionLoader;
+import io.jmix.flowui.model.DataContext;
+import io.jmix.flowui.view.*;
+import io.jmix.uisamples.entity.Food;
+import io.jmix.uisamples.view.entity.food.FoodDetailView;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+@ViewController("virtual-list-inline-editor")
+@ViewDescriptor("virtual-list-inline-editor.xml")
+public class VirtualListInlineEditor extends StandardView {
+    private static final String NOOP_DESCRIPTION = "";
+
+    @Autowired
+    private Resources resources;
+    @Autowired
+    private Messages messages;
+    @Autowired
+    private DialogWindows dialogWindows;
+
+    @ViewComponent
+    private DataContext dataContext;
+    @ViewComponent
+    private CollectionContainer<Food> foodDc;
+    @ViewComponent
+    private CollectionLoader<Food> foodDl;
+
+    @Subscribe
+    public void onBeforeShow(final BeforeShowEvent event) {
+        foodDl.load();
+    }
+
+    @Supply(to = "foodList", subject = "renderer")
+    private Renderer<Food> foodListRenderer() {
+        return new ComponentRenderer<>(item -> {
+            HorizontalLayout rootCardLayout = new HorizontalLayout();
+            rootCardLayout.setMargin(true);
+            rootCardLayout.setSpacing(true);
```

Comment:
Spacing is `true` by default??
![image](https://github.com/jmix-framework/jmix-ui-samples-2/assets/75516667/ceab5e55-9742-4dc5-84be-acc782caa08c)